docs/design/README.md

```diff
@@ -1,20 +1,26 @@
 # Dashboard Design Guide
+
```

Comment:
This can actually be based on preference and we cannot accept such changes. One empty line is allowed and we have no alignment within all files in the repo there.

Reply:
thanks for your guidance